package.json

```
@@ -15,6 +15,7 @@
     "@typescript-eslint/eslint-plugin": "4.22.1",
     "@typescript-eslint/parser": "4.22.1",
     "axios": "0.21.1",
+    "bootstrap": "^5.0.1",
```

@gutakk Can we use the specific version here?
```suggestion
    "bootstrap": "5.0.1",
```

---

Good catch. Fixed in 29c4f20f32035e12d3288aa64cebf485d012d326